# HG changeset patch
# User Rik <rik@octave.org>
# Date 1418772987 28800
#      Tue Dec 16 15:36:27 2014 -0800
# Node ID cbce5d1bcaf9429c9b36113c4669668f379d2d31
# Parent  c2f4f6eb590749f675b3b975e34d37e2d79e0558
doc: Periodic grammarcheck of documentation.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -1338,23 +1338,23 @@ struct2hdl (fig_struct);
 * Text Properties::
 * Image Properties::
 * Patch Properties::
 * Surface Properties::
 @end menu
 
 In this Section the graphics object properties are discussed in detail, starting
 with the root figure properties and continuing through the objects
-hierarchy. The documentation about a specific graphics object can be displayed 
-using @code{doc} function, e.g. @code{doc ("axes properties")} will show 
+hierarchy.  The documentation about a specific graphics object can be displayed 
+using @code{doc} function, e.g., @code{doc ("axes properties")} will show 
 @ref{Axes Properties}. 
 
 The allowed values for radio (string) properties can be retrieved 
 programmatically or displayed using the one or 
-two arguments call to @code{set} function. @xref{XREFset, , set}.
+two arguments call to @code{set} function.  @xref{XREFset, , set}.
 
 In the following documentation, default values are enclosed in @{ @}.
 
 @node Root Figure Properties
 @subsubsection Root Figure Properties
 @cindex root figure properties
 
 The @code{root figure} properties are:
@@ -1563,19 +1563,21 @@ Line styles are specified by the followi
 May be one of
 
 @table @asis
 @item @qcode{"-"}
 Solid line.  [default]
 
 @c Special handling required to avoid '--' becoming single en-dash in Info
 @ifnottex
+
 @item @verb{|"--"|}
 @end ifnottex
 @iftex
+
 @item @code{"--"}
 @end iftex
 Dashed line.
 
 @item @qcode{":"}
 Dotted line.
 
 @item @qcode{"-."}
diff --git a/libinterp/corefcn/debug.cc b/libinterp/corefcn/debug.cc
--- a/libinterp/corefcn/debug.cc
+++ b/libinterp/corefcn/debug.cc
@@ -1544,17 +1544,17 @@ Return true if in debugging mode, otherw
   else
     print_usage ();
 
   return retval;
 }
 
 DEFUN (db_next_breakpoint_quiet, args, ,
        "-*- texinfo -*-\n\
-@deftypefn {Built-in Function} {} db_next_breakpoint_quiet ()\n\
+@deftypefn  {Built-in Function} {} db_next_breakpoint_quiet ()\n\
 @deftypefnx {Built-in Function} {} db_next_breakpoint_quiet (@var{flag})\n\
 Disable line info printing at the next breakpoint.  With a logical\n\
 argument, set the state on or off.\n\
 @end deftypefn")
 {
   octave_value retval;
 
   int nargin = args.length ();
diff --git a/libinterp/corefcn/pr-output.cc b/libinterp/corefcn/pr-output.cc
--- a/libinterp/corefcn/pr-output.cc
+++ b/libinterp/corefcn/pr-output.cc
@@ -3984,18 +3984,18 @@ integer types).\n\
 @itemx plus @var{chars}\n\
 Print a @samp{+} symbol for matrix elements greater than zero, a\n\
 @samp{-} symbol for elements less than zero and a space for zero matrix\n\
 elements.  This format can be very useful for examining the structure\n\
 of a large sparse matrix.\n\
 \n\
 The optional argument @var{chars} specifies a list of 3 characters to use\n\
 for printing values greater than zero, less than zero and equal to zero.\n\
-For example, with the @samp{\"+\" \"+-.\"} format, @code{[1, 0, -1; -1, 0, 1]}\n\
-is displayed as\n\
+For example, with the @samp{\"+\" \"+-.\"} format,\n\
+@code{[1, 0, -1; -1, 0, 1]} is displayed as\n\
 \n\
 @example\n\
 @group\n\
 ans =\n\
 \n\
 +.-\n\
 -.+\n\
 @end group\n\
diff --git a/libinterp/dldfcn/symrcm.cc b/libinterp/dldfcn/symrcm.cc
--- a/libinterp/dldfcn/symrcm.cc
+++ b/libinterp/dldfcn/symrcm.cc
@@ -421,18 +421,18 @@ closer to the diagonal than @var{S}.  Th
 or Cholesky@tie{}factorization of matrices that come from ``long, skinny''\n\
 problems.  It works for both symmetric and asymmetric @var{S}.\n\
 \n\
 The algorithm represents a heuristic approach to the NP-complete\n\
 bandwidth minimization problem.  The implementation is based in the\n\
 descriptions found in\n\
 \n\
 @nospell{E. Cuthill, J. McKee}. @cite{Reducing the Bandwidth of Sparse\n\
-Symmetric Matrices}. Proceedings of the 24th ACM National Conference, 157--172\n\
-1969, Brandon Press, New Jersey.\n\
+Symmetric Matrices}. Proceedings of the 24th ACM National Conference,\n\
+157--172 1969, Brandon Press, New Jersey.\n\
 \n\
 @nospell{A. George, J.W.H. Liu}. @cite{Computer Solution of Large Sparse\n\
 Positive Definite Systems}, Prentice Hall Series in Computational\n\
 Mathematics, ISBN 0-13-165274-5, 1981.\n\
 \n\
 @seealso{colperm, colamd, symamd}\n\
 @end deftypefn")
 {
diff --git a/scripts/general/flip.m b/scripts/general/flip.m
--- a/scripts/general/flip.m
+++ b/scripts/general/flip.m
@@ -25,33 +25,27 @@
 ## Return a copy of @var{x} flipped about the dimension @var{dim}.
 ## @var{dim} defaults to the first non-singleton dimension.
 ## For example:
 ##
 ## @example
 ## @group
 ## flip ([1  2  3  4])
 ##       @result{}  4  3  2  1
-## @end group
 ##
-## @group
 ## flip ([1; 2; 3; 4])
 ##       @result{}  4
 ##           3
 ##           2
 ##           1
-## @end group
 ##
-## @group
 ## flip ([1 2; 3 4])
 ##       @result{}  3  4
 ##           1  2
-## @end group
 ##
-## @group
 ## flip ([1 2; 3 4], 2)
 ##       @result{}  2  1
 ##           4  3
 ## @end group
 ## @end example
 ##
 ## @seealso{fliplr, flipud, rot90, rotdim, permute, transpose}
 ## @end deftypefn
diff --git a/scripts/general/flipdim.m b/scripts/general/flipdim.m
--- a/scripts/general/flipdim.m
+++ b/scripts/general/flipdim.m
@@ -18,17 +18,17 @@
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} flipdim (@var{x})
 ## @deftypefnx {Function File} {} flipdim (@var{x}, @var{dim})
 ## Flip array across specific dimension.
 ##
 ## This function is an alias for @code{flip} and exists for backwards
-## and @sc{Matlab} compatibility.  See further details about its usage
+## and @sc{matlab} compatibility.  See further details about its usage
 ## on @code{flip} help text.
 ##
 ## @seealso{flip, fliplr, flipud, rot90, rotdim}
 ## @end deftypefn
 
 ## Author: David Bateman, Jaroslav Hajek
 
 function y = flipdim (varargin)
diff --git a/scripts/miscellaneous/gzip.m b/scripts/miscellaneous/gzip.m
--- a/scripts/miscellaneous/gzip.m
+++ b/scripts/miscellaneous/gzip.m
@@ -20,17 +20,17 @@
 ## @deftypefn  {Function File} {@var{filelist} =} gzip (@var{files})
 ## @deftypefnx {Function File} {@var{filelist} =} gzip (@var{files}, @var{dir})
 ## Compress the list of files and directories specified in @var{files}.
 ##
 ## @var{files} is a character array or cell array of strings.  Shell
 ## wildcards in the filename such as @samp{*} or @samp{?} are accepted and
 ## expanded.  Each file is compressed separately and a new file with a
 ## @file{".gz"} extension is created.  The original files are not modified,
-## but existing compressed files will be silently overwritten. If a directory
+## but existing compressed files will be silently overwritten.  If a directory
 ## is specified then @code{gzip} recursively compresses all files in the
 ## directory.
 ##
 ## If @var{dir} is defined the compressed files are placed in this directory,
 ## rather than the original directory where the uncompressed file resides.
 ## If @var{dir} does not exist it is created.
 ##
 ## The optional output @var{filelist} is a list of the compressed files.
diff --git a/scripts/miscellaneous/license.m b/scripts/miscellaneous/license.m
--- a/scripts/miscellaneous/license.m
+++ b/scripts/miscellaneous/license.m
@@ -25,17 +25,17 @@
 ## @deftypefnx {Function File} {@var{retval} =} license ("inuse")
 ## @deftypefnx {Function File} {@var{retval} =} license ("test", @var{feature})
 ## @deftypefnx {Function File} {@var{retval} =} license ("checkout", @var{feature})
 ## @deftypefnx {Function File} {[@var{retval}, @var{errmsg}] =} license ("checkout", @var{feature})
 ## Get license information for Octave and Octave packages.
 ##
 ## GNU Octave is free software distributed under the GNU General Public
 ## License (GPL), and a license manager makes no sense.  This function is
-## provided only for @sc{Matlab} compatibility.
+## provided only for @sc{matlab} compatibility.
 ##
 ## When called with no extra input arguments, it returns the Octave license,
 ## otherwise the first input defines the operation mode and must be one of
 ## the following strings: @code{inuse}, @code{test}, and @code{checkout}.
 ## The optional @var{feature} argument can either be @qcode{"octave"} (core),
 ## or an Octave package.
 ##
 ## @table @asis
diff --git a/scripts/miscellaneous/open.m b/scripts/miscellaneous/open.m
--- a/scripts/miscellaneous/open.m
+++ b/scripts/miscellaneous/open.m
@@ -13,27 +13,29 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{output} =} open @var{file}
-## @deftypefnx  {Function File} {@var{output} =} open (@var{file})
+## @deftypefnx {Function File} {@var{output} =} open (@var{file})
 ## Open the file @var{file} in Octave or in an external application
 ## based on the file type as determined by the file name extension.
 ##
 ## Recognized file types are
 ##
 ## @table @code
 ## @item .m
 ## Open file in the editor.
+##
 ## @item .mat
 ## Load the file in the base workspace.
+##
 ## @item .exe
 ## Execute the program (on Windows systems only).
 ## @end table
 ##
 ## Other file types are opened in the appropriate external application.
 ## @end deftypefn
 
 function output = open (file)
diff --git a/scripts/miscellaneous/perl.m b/scripts/miscellaneous/perl.m
--- a/scripts/miscellaneous/perl.m
+++ b/scripts/miscellaneous/perl.m
@@ -19,17 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{output} =} perl (@var{scriptfile})
 ## @deftypefnx {Function File} {@var{output} =} perl (@var{scriptfile}, @var{argument1}, @var{argument2}, @dots{})
 ## @deftypefnx {Function File} {[@var{output}, @var{status}] =} perl (@dots{})
 ## Invoke Perl script @var{scriptfile}, possibly with a list of command line
 ## arguments.
 ##
 ## Return output in @var{output} and optional status in @var{status}.  If
-## @var{scriptfile} is not an absolute file name it is is searched for in the
+## @var{scriptfile} is not an absolute file name it is searched for in the
 ## current directory and then in the Octave loadpath.
 ## @seealso{system, python}
 ## @end deftypefn
 
 function [output, status] = perl (scriptfile = "-e ''", varargin)
 
   ## VARARGIN is intialized to {}(1x0) if no additional arguments are
   ## supplied, so there is no need to check for it, or provide an
diff --git a/scripts/miscellaneous/python.m b/scripts/miscellaneous/python.m
--- a/scripts/miscellaneous/python.m
+++ b/scripts/miscellaneous/python.m
@@ -20,17 +20,17 @@
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{output} =} python (@var{scriptfile})
 ## @deftypefnx {Function File} {@var{output} =} python (@var{scriptfile}, @var{argument1}, @var{argument2}, @dots{})
 ## @deftypefnx {Function File} {[@var{output}, @var{status}] =} python (@dots{})
 ## Invoke Python script @var{scriptfile}, possibly with a list of command line
 ## arguments.
 ##
 ## Return output in @var{output} and optional status in @var{status}.  If
-## @var{scriptfile} is not an absolute file name it is is searched for in the
+## @var{scriptfile} is not an absolute file name it is searched for in the
 ## current directory and then in the Octave loadpath.
 ## @seealso{system, perl}
 ## @end deftypefn
 
 ## Author: Carnë Draug <carandraug+dev@gmail.com>
 
 function [output, status] = python (scriptfile = "-c ''", varargin)
 
diff --git a/scripts/miscellaneous/recycle.m b/scripts/miscellaneous/recycle.m
--- a/scripts/miscellaneous/recycle.m
+++ b/scripts/miscellaneous/recycle.m
@@ -20,19 +20,20 @@
 ## @deftypefn  {Function File} {@var{current_state} =} recycle ()
 ## @deftypefnx {Function File} {@var{old_state} =} recycle (@var{new_state})
 ## Query or set the preference for recycling deleted files.
 ##
 ## When recycling is enabled, commands which would permanently erase files
 ## instead move them to a temporary location (such as the directory labeled
 ## Trash).
 ##
-## Programming Note: This function is provided for Matlab compatibility, but
-## recycling is not implemented in Octave.  To help avoid accidental data loss
-## an error will be raised if an attempt is made to enable file recycling.
+## Programming Note: This function is provided for @sc{matlab} compatibility,
+## but recycling is not implemented in Octave.  To help avoid accidental data
+## loss an error will be raised if an attempt is made to enable file
+## recycling.
 ## @seealso{delete, rmdir}
 ## @end deftypefn
 
 ## Author: jwe
 
 function retval = recycle (state)
 
   persistent current_state = "off";
diff --git a/scripts/miscellaneous/what.m b/scripts/miscellaneous/what.m
--- a/scripts/miscellaneous/what.m
+++ b/scripts/miscellaneous/what.m
@@ -25,32 +25,41 @@
 ## If @var{dir} is not specified then the current directory is used.
 ##
 ## If a return argument is requested, the files found are returned in the
 ## structure @var{w}.  The structure contains the following fields:
 ##
 ## @table @asis
 ## @item path
 ## Full path to directory @var{dir}
+##
 ## @item m
 ## Cell array of m-files
+##
 ## @item mat
 ## Cell array of mat files
+##
 ## @item mex
 ## Cell array of mex files
+##
 ## @item oct
 ## Cell array of oct files
+##
 ## @item mdl
 ## Cell array of mdl files
+##
 ## @item slx
 ## Cell array of slx files
+##
 ## @item p
 ## Cell array of p-files
+##
 ## @item classes
 ## Cell array of class directories (@file{@@@var{classname}/})
+##
 ## @item packages
 ## Cell array of package directories (@file{+@var{pkgname}/})
 ## @end table
 ##
 ## Compatibility Note: Octave does not support mdl, slx, and p files; nor
 ## does it support package directories.  @code{what} will always return an
 ## empty list for these categories.
 ## @seealso{which, ls, exist}
diff --git a/scripts/sparse/ichol.m b/scripts/sparse/ichol.m
--- a/scripts/sparse/ichol.m
+++ b/scripts/sparse/ichol.m
@@ -51,18 +51,18 @@
 ## @item diagcomp
 ## A non-negative scalar @var{alpha} for incomplete Cholesky factorization of
 ## @code{@var{A} + @var{alpha} * diag (diag (@var{A}))} instead of @var{A}.
 ## This can be useful when @var{A} is not positive definite.  The default value
 ## is 0.
 ##
 ## @item droptol
 ## A non-negative scalar specifying the drop tolerance for factorization if
-## performing @nospell{ICT}@.  The default value is 0 which produces the complete
-## Cholesky factorization.
+## performing @nospell{ICT}@.  The default value is 0 which produces the
+## complete Cholesky factorization.
 ##
 ## Non-diagonal entries of @var{L} are set to 0 unless
 ##
 ## @code{abs (@var{L}(i,j)) >= droptol * norm (@var{A}(j:end, j), 1)}.
 ##
 ## @item michol
 ## Modified incomplete Cholesky factorization:
 ##
